CodeForBaltimore/Bmore-Responsive
# Fix null deletion reference

# Description

- Added null check for deletion
- Updated formatting

Resolves #341 

## Todos


## Deploy Notes

- N/A
- 
## Steps to Test or Reproduce

1. Verify that Postman tests do not fail on repeated runs
2. Verify that 500 is not returned even when there are no links

## Impacted Areas in Application

List general components of the application that this PR will affect:

- Contact unlinking from one or more entities
